pkg/systemd/overview-cards/uncleanShutdownStatus.scss

```
@@ -0,0 +1,8 @@
+.system-information-unclean-shutdown-status-icon {
+  color: var(--pf-t--global--color--status--danger--100);
+}
+
+.system-information-unclean-shutdown-status {
+  color: var(--pf-t--global--color--status--danger--200);
+  font-weight: var(--pf-t--global--font--weight--body--bold);
+}
```

Don't think we need to change font-weight here either
```suggestion
.system-information-unclean-shutdown-status {
  color: var(--pf-t--global--text--color--status--danger--default);
}
```